client/src/components/application/ResponsiveDrawer/list.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { loadCSS } from 'fg-loadcss';
+import Icon from '@material-ui/core/Icon';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+const DrawerList = ({ content, isFinished }) => {
```

Comment:
Instead of having two arrays that need to be the same length and match one another, it might be simpler to have one array and change its shape to have both text and isFinished inside of each item.  So for example, content would look like `[{text: 'first one', isFinished: false}, {text: 'second one', isFinished: true}, {text: 'third thing!', isFinished: true}]`